scripts/build_catkin_workspace.sh

```diff
@@ -71,14 +74,18 @@ done
 # Abort script on any failures
 set -e
 
-if [ "$PREFIX_PATH" = "" ]; then
-  print_help
-  exit 1
+if [ "$OUTPUT_PREFIX" = "" ]; then
+  if ["$OUTPUT_DIR" = ""]; then
+    print_help
+    exit 1
+  else
+    OUTPUT_PREFIX=$OUTPUT_DIR
+  fi
```

Comment:
nit: Is there a good reason to use two separate variables `OUTPUT_DIR` and `OUTPUT_PREFIX`? I understand that one of them can be set by the user and the other is an internal variable, but it would be possible to use only one and eventually overwrite it if specified in the command line, no?

Reply:
absolutely right.